packages/admin-ui/ui/src/components/forms/gift-card/gift-card-ends-at-form/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Controller } from "react-hook-form"
```

Comment:
todo: Adding/updating `ends_at` triggers a form submit

![CleanShot 2023-04-02 at 11 14 47](https://user-images.githubusercontent.com/59018053/229343752-754b272e-9f1d-4a31-a94d-27719405fdf0.gif)

Reply:
great catch the number selector component was defaulting its buttons to submit type so this was a problem in multiple forms.